_data/people.yml

```diff
@@ -290,6 +267,32 @@ zeeshan:
     display_name: "Zeeshan Ahmad"
     role: alumni-phd
     bio: Postdoctoral Research Scholar at University of Chicago
+    webpage: https://www.andrew.cmu.edu/user/azeeshan/
+
+greg:
+    display_name: "Gregory Houchins"
+    role: alumni-phd
+    image: img/people/Gregory_Houchins.png
+    webpage: "https://andrew.cmu.edu/~ghouchin"
+    email: "ghouchin [at] andrew [dot] cmu [dot] edu"
+    bio: Swartz Center for Entrepreneurship Innovation Fellow, Carnegie Mellon University
+
+Dilip:
+    display_name: "Dilip Krishnamurthy"
+    role: alumni-phd
+    image: img/people/dilip_headshot.jpg
+    bio: Mechanical Engineering
+    email: dkrishn1 [at] andrew [dot] cmu [dot] edu
+    webpage: "https://dilipkrishnamurthy.github.io/"
+
+olga:
+    display_name: "Olga Vinogradova"
+    role: alumni-phd
+    image: img/people/Olga_Vinogradova.jpg
+    bio: Chemical Engineering
+    email: ovinogra [at] andrew [dot] cmu [dot] edu
```

Comment:
bio: Postdoctoral Researcher at Fritz Haber Institute of the Max Planck Society
email: ovinogra [at] alumni [dot] cmu [dot] edu

Reply:
Please change the relevant lines to this